testsuite/tests/capsule-api/key.ml

```diff
@@ -0,0 +1,71 @@
+(* TEST
+ include stdlib_alpha;
+ flags = "-extension-universe alpha";
+ { bytecode; }
+ { native; }
+*)
+
+module Capsule = Stdlib_alpha.Capsule
+
+external ( = ) : 'a -> 'a -> bool @@ portable = "%equal"
+
+external raise : exn -> 'a @ portable unique @@ portable = "%reraise"
+
+type 'a aliased = Aliased of 'a @@ aliased [@@unboxed]
+
+
+
+let () =
+  (* [Capsule.create] creates a new capsule with an unique key. *)
+  let packed : Capsule.Key.packed @@ unique = Capsule.create () in
+  let (P k) = packed in
+  (* [Capsule.Key.access] *)
+  let Aliased x, k =
+    Capsule.Key.access k (fun a ->
+      Aliased (Capsule.Data.wrap a "Hello world!"))
```

Comment:
It's probably better to use a mutable type for the contents of the capsule, rather than `string`.